rustler/src/wrapper/pid.rs

```diff
@@ -16,3 +16,20 @@ pub unsafe fn get_local_pid(env: NIF_ENV, term: NIF_TERM) -> Option<ErlNifPid> {
 pub unsafe fn make_pid(env: NIF_ENV, pid: ErlNifPid) -> NIF_TERM {
     rustler_sys::enif_make_pid(env, pid)
 }
+
+///
+/// Look up a local process by its registered name.
+///
+/// # Safety
+///
+/// `env` must only be null if the calling process is a created process. Otherwise,
```

Comment:
The [section](http://erlang.org/doc/man/erl_nif.html#enif_whereis_pid) in the documentation reads:

> The environment of the calling process. Must be NULL only if calling from a created thread.

Does that mean that it **MUST** be `NULL` if calling from a created thread, or that it **MAY** be `NULL` in that case?

Reply:
I created an issue on the erlang issue tracker for this.

https://bugs.erlang.org/browse/ERL-1399

I think they mean it in the strictest sense, but would be nice to get confirmation about this.

Reply:
> It's the first: NULL for created thread, otherwise the environment argument (process bound environment or callback environment).